docs/sections/getting_started.md

```diff
@@ -1,8 +1,8 @@
 # Getting Started
 
-Popper is a workflow execution engine based on [Github Actions][gha] 
-(GHA) written in Python. With Popper, you can execute [HCL 
-syntax][ghalang] workflows locally on your machine without having 
+Popper is a workflow execution engine based on [Github Actions][gha]
```

Comment:
looks like this PR also contains the commits from the other PR you're also working on. Can you please remove them from this? or alternatively add both in a single PR and close one of them?

Reply:
Alright I'd do exactly this